generator/README.md

```diff
@@ -43,6 +43,11 @@ docker run -ti \
   snmp-generator generate
 ```
 
+Please note: In CI we don't run the generator with the MIBs in `/usr/share/snmp`
```

Comment:
Yea, I think we should use https://github.com/prometheus/snmp_exporter/pull/385 to make it work by default.

Reply:
Aye, I'll remove that.